js/utility/DltBtnAction.js

```diff
@@ -0,0 +1,32 @@
+import { CLICK_EVENT, EDIT, COMPLETE, DELETE_TODO } from "/js/constants.js";
+
+export default function handleDeleteButtonClick(
+  todoItemElement,
+  saveButtonElement,
+  deleteButtonElement,
+  completeButtonElement
+) {
+  const previousContent = todoItemElement.querySelector(".text").innerText;
+
+  function handleDeleteButton(event) {
+    event.stopPropagation();
+    event.preventDefault();
+    const errorMessage = todoItemElement.querySelector("#error-message");
+    if (errorMessage) {
+      todoItemElement.removeChild(errorMessage);
+    }
+    todoItemElement.querySelector(".text").contentEditable = false;
+    todoItemElement.querySelector(".text").innerText = previousContent;
+    todoItemElement.querySelector(`[data-job="${EDIT}"]`).style.display =
+      "inline-block";
+    todoItemElement.querySelector(`[data-job="${DELETE_TODO}"]`).style.display =
+      "inline-block";
+    todoItemElement.querySelector(`[data-job="${COMPLETE}"]`).style.display =
+      "inline-block";
```

Comment:
```suggestion
    [EDIT, DELETE_TODO, COMPLETE].forEach(jobType =>  todoItemElement.querySelector(`[data-job="${jobType}"]`).style.display =
      "inline-block")
```

Reply:
same goes for [displayButton](https://github.com/ban-wtag/ToDoAppJS/pull/5#discussion_r1182194149)